gantry/util/k8s.py

```diff
@@ -0,0 +1,14 @@
+CORES_TO_MILLICORES = 1000
+BYTES_TO_MEGABYTES = 1 / 1_000_000
+
+# these functions convert the predictions to k8s friendly format
+
+
+def convert_cores(cores: float) -> str:
+    """cores to millicores"""
+    return str(int(cores * CORES_TO_MILLICORES)) + "m"
```

Comment:
Let's talk with @tgamblin, @scottwittenburg, and @jjnesbitt -- It'd be great for understandability of the system if we could get away from using partial CPU values.

Reply:
CPU units are now cores rather than millicores and the predictions are rounded up